code/game/objects/items/devices/taperecorder.dm

```diff
@@ -310,6 +310,20 @@
 	canprint = FALSE
 	addtimer(VARSET_CALLBACK(src, canprint, TRUE), 30 SECONDS)
 
+/obj/item/taperecorder/verb/WipeTapeInRecorder()
+	set name = "Wipe Tape"
+
+	if(!mytape || mytape.unspooled)
+		return
+	if(recording)
+		return
+	if(playing)
+		return
```

Comment:
you could combine this into a single if statement like you did with the previous one
```suggestion
	if(recording || playing)
	    return FALSE
```

Reply:
Oh, fuck, that's so good. Why did I not fucking think of this?